go.mod

```diff
@@ -1,9 +1,85 @@
 module github.com/Juniper/junos-terraform
 
-go 1.14
+go 1.17
 
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/davedotdev/go-netconf v0.1.3
-	github.com/hashicorp/terraform-plugin-sdk v1.16.1
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.16.0
+)
+
+require (
+	cloud.google.com/go v0.65.0 // indirect
```

Comment:
What is the License for each of these new modules that are added as dependencies?

Reply:
I have removed the Indirect dependency. User will be prompted to do `go mod tidy` to update these before doing go build.